### ISSUE TITLE
Using another node than https://test.massa.net/api/v2 

### ISSUE THREAD

**Issue by gregLibert** [massalabs/station#483]
**Describe the bug**
I have the following error message in my browser console:
```json
{"code":"EventListener-0001","message":"calling endpoint get_filtered_sc_output_event with '[{Start:0xc0010840c0 End:<nil> EmitterAddress:0xc000459628 OriginalCallerAddress:<nil> OriginalOperationID:<nil>}]': rpc call get_filtered_sc_output_event() on https://test.massa.net/api/v2 status code: 503. could not decode body to rpc response: invalid character '<' looking for beginning of value"}
```

**To Reproduce**
I updated a website and I'm stuck at the upload chunk step. At some point one I have the previous message in the console.

**Expected behavior**
The first expectation is to have the website to be updated. Or at least to have visibility, but there are already multiple issues on that (#260, #281, #349).
The second one is linke to my understanding of the process:
the error message is talking about an invalid character (<) because the call is returning HTTP. And if HTTP is returned it's because the proxy is filtering the request and at some point refusing some.

I have then two suggestions:
- use other servers than test.massa.net/api/v2
- detect such proxy rejections and automatically retry.

**Screenshots**
I shared the error message.

**Thyra version**
baedfc7675e131f0ad87991424ccf579adf70997

**Desktop (please complete the following information):**
 - OS: linux ubuntu
 - Browser: chrome
 - Version: ?

**Additional context**
Nothing else to share.

**Comment by gregLibert** [comment]
Another error: 
```bash
2023/02/28 13:31:19 Could not get node version: calling get_status: rpc call get_status() on https://test.massa.net/api/v2 status code: 502. could not decode body to rpc response: invalid character '<' looking for beginning of value
2023/02/28 13:31:19 Connected to node server https://test.massa.net/api/v2 (version unknown)
```
but having the same source.

**Comment by SlnPons** [comment]
@peterjah have we address this problem now we can configure a node connection using the CLI? 
Or am I dreaming and we haven't done that yet? 😬

**Comment by SlnPons** [comment]
That would be great to extend this reported issue to enable end-users to connect Massa Station to the node of their choice very easily using the FE. 
This use case needs to be discussed and solution must be refined with Pierre S before addressing this ticket, please make sure to do so :)

**Comment by longgamma-massa** [comment]
@peterjah ping me to sync on this

**Comment by gregLibert** [comment]
Included in #279
